Mathlib/Algebra/Exact.lean

```diff
@@ -90,6 +90,42 @@ lemma exact_of_comp_of_mem_range
     (h1 : g.comp f = 0) (h2 : ∀ x, g x = 0 → x ∈ range f) : Exact f g :=
   exact_of_comp_eq_zero_of_ker_le_range h1 h2
 
+/-- When we have a commutative diagram from a sequence of two maps to another,
+such that the left map is surjective, the middle map is bijective and the right
+map is injective, then the upper row is exact iff the lower row is. -/
```

Comment:
```suggestion
such that the left vertical map is surjective, the middle vertical map is bijective and the right vertical
map is injective, then the upper row is exact iff the lower row is. -/
```

Reply:
Maybe also link to `CategoryTheory.ShortComplex.exact_iff_of_epi_of_isIso_of_mono` here? (and below)